python/housinginsights/sources/cama.py

```diff
@@ -0,0 +1,208 @@
+from pprint import pprint
+import os
+import sys
+import requests
+from collections import OrderedDict
+import csv
+import datetime
+
+PYTHON_PATH = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir, os.pardir))
+sys.path.append(PYTHON_PATH)
+
+from housinginsights.sources.base import BaseApiConn
+from housinginsights.sources.models.mar import MarResult, FIELDS
+
+
+class MarApiConn(BaseApiConn):
```

Comment:
Note that we already have a MarApiConn class in the sources/mar.py file . I suggest renaming this one for now (e.g. SecondMarApiConn), and during your refactor see if you can get it to just use one single MarApiConn class.